setup.py

```diff
@@ -669,13 +654,6 @@ def run(self):
         return install_data.run(self)
 
 
-@add_command('sdist')
```

Comment:
I'd prefer MANIFEST.in to stay in the buildconfig/ folder.

Reply:
I'm OK with reverting this and fixes the actual bug in the function, but my reasons against this are (as stated in c3c6abce0c07a99b6e2a3231952ccac3573f14e1):

1. The others files in the project's root directory are only README and setuptools' configuration (slightly unrelated: setup.cfg happens to store linters' config, which IMHO should be moved to e.g. tox.ini), which makes sense for MANIFEST.in to be in the same dir.
2. Somehow (I can't find this documented) with this specified, setuptools will write MANIFEST to the root directory.

Reply:
We want to try and make it as easy as possible for people to contribute. Things like MANIFEST.in aren't useful for 95% of changes people make to pygame, for that reason it's best to hide it away in the buildconfig/ folder. If there was someway the MANIFEST.in could be made to not be written, that would be good. But I guess it should be moved into a separate PR?

The reason for the setup.cfg file is that all python tools can be configured in there. Including pylint, mypy, and others we want to make more use of.

Reply:
Thanks, I'll address the generation of `MANIFEST` later.